sites/shared/components/workbench/menu/core-settings/core-setting-sa-mm.mjs

```diff
@@ -5,14 +5,29 @@ import { useTranslation } from 'next-i18next'
 
 export const CoreSettingSaMm = (props) => {
   const { t } = useTranslation(['app', 'settings'])
-  const { dflt, min, max } = props
+  const { dflt, min, max, step, dfltIm, maxIm, stepIm } = props
   const val = props.gist?.[props.setting]
 
+  let dfltVal
+  let maxVal
+  let stepVal
+  if (props.gist.units == 'imperial') {
+    dfltVal = dfltIm
+    maxVal = maxIm
+    stepVal = stepIm
+  } else {
+    dfltVal = dflt
+    maxVal = max
+    stepVal = step
+  }
+
   const [value, setValue] = useState(val)
 
   const handleChange = (evt) => {
     const newVal = parseFloat(evt.target.value)
 
+    setValue(stepVal)
```

Comment:
Was this added line intended? 

I tested the PR in the lab with and without the line, and I wasn't able to figure out what the line was doing. The lab seemed to work fine either way.

Reply:
Yea Im not really sure where that came from. I doubled check and yep it does nothing so I have removed it. Not really sure why I added that? Seems like I may have done that on auto-pilot *shrugs*